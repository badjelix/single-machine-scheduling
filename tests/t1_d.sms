10
0 2 2 1 2
9 2 11 2 1 1
9 3 15 1 3
6 1 7 1 1
15 12 51 3 3 4 5
2 4 14 3 1 2 1
6 2 10 2 1 1
17 7 38 5 1 1 2 2 1
1 6 7 4 1 1 2 2
7 12 19 3 4 3 5
0
0
1 6
0
2 6 4
0
0
2 4 5
0
0
