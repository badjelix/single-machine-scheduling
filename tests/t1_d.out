8
1 0
2 9 10
3 11
4 6
5 17 22 26
6 2 3 5
7 7 8
8 31 32 33 35 37
